redaxo/src/core/lib/Fragment/Enum/Placement.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Redaxo\Core\Fragment\Enum;
+
+enum Placement: string
```

Comment:
ggf. nen präzisieren namen wählen?

die werte scheinen für spezifische use-cases zu sein, daher ggf. nicht ganz so allgemein bennen?

Reply:
hier sollten wir denke ich auch spezifischer sein, da es auch fürs popup/tooltip ein placement gibt, was andere values zulässt.

https://shoelace.style/components/popup#placement
https://shoelace.style/components/tooltip#placement

Reply:
ich würde die enums genauso strukturieren wie in shoelace